packages/utils/src/ss58Format.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * Copyright (c) 2018-2022, BOTLabs GmbH.
+ *
+ * This source code is licensed under the BSD 4-Clause "Original" license
+ * found in the LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * The network SS58 prefix registered for KILT blockchain
```

Comment:
Do we also need something like @packageDocumentation here?

Reply:
not sure. But we will have to go through the API documentation either way at the end, because some package documentations might already not work anymore.